.ocr/commands/doctor.md

```
@@ -0,0 +1,37 @@
+---
+description: Check OCR installation and verify all dependencies are available.
+name: "OCR: Doctor"
+category: Code Review
+tags: [ocr, setup, diagnostics]
+---
+
+**Usage**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)